website/src/release.md

```diff
@@ -316,33 +320,33 @@ After downloading them, here are the instructions on how to verify them.
 curl https://downloads.apache.org/iceberg/KEYS -o KEYS
 gpg --import KEYS
 ```
-* Verify the `.asc` file: ```gpg --verify apache-iceberg-rust-${iceberg_version}.tar.gz.asc```
-* Verify the checksums: ```shasum -a 512 -c apache-iceberg-rust-${iceberg_version}.tar.gz.sha512```
-* Verify license headers: ```docker run -it --rm -v $(pwd):/github/workspace apache/skywalking-eyes header check```
+* Verify the `.asc` file: ```gpg --verify apache-iceberg-rust-*.tar.gz.asc```
```

Comment:
An expected output would be helpful, I think we basically expect a line like this
```
gpg: Good signature from XXX
```

Reply:
added some expectations

<img width="867" height="900" alt="Screenshot 2025-09-18 at 7 41 14 AM" src="https://github.com/user-attachments/assets/48e47899-f2bb-4726-afa2-230969e19e36" />